src/Content/Backend/Solution/Monaco.Template.Backend.Api/Controllers/FilesController.cs

```diff
@@ -12,26 +12,20 @@
 #endif
 using Microsoft.AspNetCore.Mvc;
 using System.Net;
+using Asp.Versioning;
 
 namespace Monaco.Template.Backend.Api.Controllers
```

Comment:
Would recommend using file scoped namespaces where possible.

Reply:
Oops, must be a leftover from some time ago that I missed... Will change it later, for sure.

Reply:
Done.